doc/nrf/ug_tfm.rst

```diff
@@ -48,18 +52,30 @@ When building for ``nrf9160dk_nrf9160ns``, UART1 must be disabled in the non-sec
 Otherwise, the non-secure application will fail to run.
 The recommended way to do this is to copy the .overlay file from the :ref:`tfm_hello_world` sample.
 
+Using TF-M with nrf_security
+============================
+
+When using the :ref:`nrfxlib:nrf_security`, if :option:`CONFIG_BUILD_WITH_TFM` is enabled together with :option:`CONFIG_NORDIC_SECURITY_BACKEND`, the TF-M secure image will enable the use of the hardware acceleration of Arm CryptoCell.
+
+In such case, the Kconfig configurations in the Nordic Security Backend control the features enabled through TF-M.
+
 .. _tfm_minimal_build:
 
 Minimal build
 =============
 
 The default configuration of TF-M has all supported features enabled, which results in a significant memory footprint.
-A minimal version of the TF-M secure application is provided to show how to configure a reduced version of TF-M.
-The non-secure callable interfaces supported by this version generating random numbers, hashing with SHA256, and `tfm_platform_mem_read`.
-The minimal version is enabled by setting the :option:`CONFIG_TFM_MINIMAL` option.
+A minimal version of the TF-M secure application is provided in |NCS| to show how to configure a reduced version of TF-M.
+
+The non-secure callable interfaces supported by this minimal version allow for generating random numbers, hashing with SHA-256, and using ``tfm_platform_mem_read``.
```

Comment:
Not sure what nomenclature we are using, but we most often refer to these interfaces as "secure services".

Reply:
will change 'non-secure callable interfaces' to 'secure services' 

is that ok with you @hakonfam